pkg/daemon/ceph/client/filesystem.go

```diff
@@ -309,6 +309,43 @@ func FailMDS(context *clusterd.Context, clusterName string, gid int) error {
 	return nil
 }
 
+// DownFilesystem marks the filesystem as down and the MDS' as failed
+func DownFilesystem(context *clusterd.Context, cephVersion cephver.CephVersion, clusterName, filesystemName string) error {
+	logger.Infof("Downing filesystem %s", filesystemName)
+
+	if cephVersion.IsAtLeastNautilus() {
+		args := []string{"fs", "fail", filesystemName}
```

Comment:
I think we should make a ceph client command for issuing `ceph fs fail` command.

Reply:
Done